cumulus_process/process.py

```diff
@@ -134,15 +134,7 @@ def upload_file(self, filename):
         if info is None:
             return filename
         try:
-            uri = None
-            if info.get('s3', None) is not None:
-                bucket = self.buckets.get(info.get('bucket', 'public'), None)
-                bucketType = None
-                if bucket is not None:
-                    bucketType = bucket['type']
-                extra = {'ACL': 'public-read'} if bucketType == 'public' else {}
-                uri = upload(filename, info['s3'], extra=extra)
-            return uri
+            return upload(filename, info['s3'], extra={}) if info.get('s3', False) else None
```

Comment:
Did you try directly using the `upload` function in `s3.py`? I'm wondering if we should switch over to that and delete this since we said we'd be removing it in the next release.

Reply:
Yes using helpers functions I could successfully upload to S3 bucket. But you will need to manually filter which bucket a file needs to go to.